libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -32,13 +32,24 @@
           <p [innerHTML]="b.description"></p>
           <div>
             <button
+              *ngIf = "!b.markedFinished else finishedButton"
+              data-testing = "mark-to-read"
```

Comment:
I would watch the consistency of spaces around the `=` sign of attributes. Preferably I would remove these spaces, like you have in your other attributes.